src/doc/unstable-book/src/language-features/intrinsics.md

```diff
@@ -2,13 +2,32 @@
 
 The tracking issue for this feature is: None.
 
-Intrinsics are never intended to be stable directly, but intrinsics are often
+Intrinsics are rarely intended to be stable directly, but are usually
 exported in some sort of stable manner. Prefer using the stable interfaces to
 the intrinsic directly when you can.
 
 ------------------------
 
 
+## Intrinsics with fallback logic
+
+Many intrinsics can be written in pure rust, albeit inefficiently or without supporting
+some features that only exist on some backends. Backends can simply not implement those
+intrinsics without causing any code miscompilations or failures to compile.
+
+```rust
+#![feature(core_intrinsics)]
+
+#[rustc_intrinsic(name_of_things)]
+fn foo() -> u32 {
```

Comment:
Since you've decided to make an example here where the function name is distinct from the name fed into the `#[rustc_intrinsic]` attribute, can you add documentation above explaining the significance of the name fed into the attribute?

Reply:
Oh never mind, I just saw 42d2fe0f7b79d6e4ab4a6cb578b092a7d03813ad

Reply:
this was just an old commit from the other PR